pre_commit/resources/hook-tmpl

```diff
@@ -8,6 +8,7 @@ popd > /dev/null
 
```

Comment:
I usually `md5sum setup.py`, but it really could be anything, just needs to be unique :)